src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -243,6 +243,20 @@ class JooqTouchstoneRepository(
         return record.id
     }
 
+    override fun getCoverageUploadMetadata(touchstoneVersionId: String): List<CoverageUploadMetadata>
+    {
+        return dsl.select(COVERAGE_SET.VACCINE,
```

Comment:
Would it be helpful for these to come out of the db in any particular order for display to user? Could by by recency of upload or maybe by vaccine would be most useful for user checking which vaccines have/haven't been uploaded?

Reply:
Yeah, maybe vaccine ordered might be most useful?